src/work-allocation/containers/task-home/task-home.component.ts

```diff
@@ -15,7 +15,230 @@ export class TaskHomeComponent implements OnInit {
    * Take in the Router so we can navigate when actions are clicked and
    * to identify which sub-navigation item to highlight.
    */
-  constructor(private readonly router: Router) {}
+  public tasks: Task[] = [
```

Comment:
Looks like a merge issue with this - it was removed in an earlier commit and isn't needed in here. :)

Reply:
Removed it now. Thanks.